libraries/AP_HAL_Linux/HAL_Linux_Class.cpp

```diff
@@ -165,6 +170,8 @@ static RCInput_Multi rcinDriver{2, NEW_NOTHROW RCInput_RCProtocol("/dev/uart-sbu
 static RCInput_SoloLink rcinDriver;
 #elif CONFIG_HAL_BOARD_SUBTYPE == HAL_BOARD_SUBTYPE_LINUX_RST_ZYNQ
 static RCInput_RCProtocol rcinDriver{"/dev/ttyPS0", NULL};
+#elif CONFIG_HAL_BOARD_SUBTYPE == HAL_BOARD_SUBTYPE_LINUX_PILOTPI
+static RCInput_Multi rcinDriver{2, NEW_NOTHROW RCInput_RCProtocol("/dev/ttyAMA0", NULL), NEW_NOTHROW RCInput_UDP()};
```

Comment:
Use AP_RCProtocol, do not use RCInput_Multi

Reply:
I don't quite get this change. I'm basically copying [how DISCO does it](https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_HAL_Linux/HAL_Linux_Class.cpp#L163) (and others using RCInput_Multi).

You mean drop `RCInput_UDP` support entirely?

Reply:
done